srcpkgs/haxor-news/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'haxor-news'
+pkgname=haxor-news
+version=0.4.2
+revision=1
+noarch=yes
+build_style=python3-module
+pycompile_module="haxor_news"
```

Comment:
`pycompile_module="haxor_news tests"`

Reply:
`tests` are outside the namespace and must be removed.